src/app/@ansyn/ansyn/modules/menu-items/layers-manager/services/data-layers.service.ts

```diff
@@ -75,7 +75,7 @@ export class DataLayersService implements OnInit, OnDestroy {
 		}
 		return this.storageService.searchByCase<ILayer>(this.config.schema, { caseId })
 			.pipe(
-				map(layers => of(layers.slice().reverse())),
+				map(layers => layers.slice().reverse()),
```

Comment:
slice is also not necessary.

Reply:
i think it takes the array as a refrence, and switches its order in the store.
wouldnt it create problems on other selections?

Reply:
@tltk90

Reply:
actually, i think the reverse() happends before the layers enter the store, so the slice() method might be redudent